README.md

```diff
@@ -59,7 +58,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
In order to make your readme file clear, I suggest you to remove all unrelevant comments and sections you didn't fill out